bin/api-docs/gen-components-docs/markdown/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+const json2md = require( 'json2md' );
+
+/**
+ * Internal dependencies
+ */
+const { generateMarkdownPropsJson } = require( './props' );
+
+function generateMarkdownDocs( { typeDocs, subcomponentTypeDocs } ) {
```

Comment:
Do we need some checks if `typeDocs` and `subcomponentTypeDocs` are defined and valid before using them later?